cluster/src/main/java/org/apache/iotdb/cluster/log/manage/serializable/SyncLogDequeSerializer.java

```diff
@@ -621,6 +650,7 @@ private void actuallyDeleteFile() {
         logger.debug("Examining file for removal, file: {}, len: {}, removedLogSize: {}", logFile
             , logFile.length(), removedLogSize);
       }
+      long len = logFile.length();
```

Comment:
Where is this used？

Reply:
It was used for debugging. I will remove it. Thanks for your reminding.